doc/src/modules/vector/vector_integration.rst

```diff
@@ -0,0 +1,144 @@
+================================
+Applications of Vector Integrals
+================================
+
+To integrate a scalar or vector field over a region, we have to first define a region. SymPy provides three methods for defining a region:
+
+1. Using Parametric Equations.
+
+2. Using Implicit Equation.
+
+3. Using objects of geometry module.
+
+The :func:`~sympy.vector.integrals.vector_integrate` function is used to integrate scalar or vector field over any type of region. It automatically determines the type of integration (line, surface or volume) depending on the nature of the object.
+
+We define a coordinate system and make necesssary imports for examples.
+>>> from sympy import sin, cos, exp, pi, symbols
+>>> from sympy.vector import CoordSys3D, ParametricRegion, ImplicitRegion, vector_integrate
+>>> from sympy.abc import r, x, y, z, theta, phi
+>>> C = CoordSys3D('C')
+    
+Calculation of Perimeter, Surface Area and Volume
+=================================================
+
+To calculate the perimeter of circle, we need to define a circle. Let's define it using its parametric equation.
+
+>>> param_circle = ParametricRegion((4*cos(theta), 4*sin(theta)), (theta, 0, 2*pi))
+
+We can also define circle using its implicit equation.
+
+>>> implicit_circle = ImplicitRegion((x, y), x**2 + y**2 - 4)
+
+Perimeter of a figure is equal to the absolute value of its integral over a unit scalar field. 
+
+>>> vector_integrate(1, param_circle)
+8*pi
+>>> vector_integrate(1, implicit_circle)
+8*pi
+
+Suppose a user wants to calculate the perimeter of triangle. Determining the parametric representation of triangle can be difficult. Instead, the user can use an object of :class:`~sympy.geometry.polygon.Polygon` class in the geometry module.
+
+>>> from sympy.geometry import Point, Polygon 
+>>> triangle = Polygon(Point(1, 2), (3, 5), (1,6))
+>>> vector_integrate(1, triangle)
+sqrt(5) + sqrt(13) + 4
+
+To define a solid sphere, we need to use three parameters (r, theta and phi). 
+
+>>> solidsphere = ParametricRegion((r*sin(phi)*cos(theta),r*sin(phi)*sin(theta), r*cos(phi)),
+... (phi, 0, pi), (theta, 0, 2*pi), (r, 0, 3))
+>>> abs(vector_integrate(1, solidsphere))
```

Comment:
is it a problem in the algorithm that you get negative results? Why is `vector_integrate` returning a negative result?

Reply:
This is due to the order of variables used in the calculation of Jacobian. The order is decided by function `bounds_case`. `bounds_case` is required for dealing with situations where bounds of variables are interdependent.

Please see my comment above on this.

Reply:
I have made a PR https://github.com/sympy/sympy/pull/20044 for this.

Reply:
OK, it's merged. Can you remove `abs( )` now?

Reply:
Yes.